vendor/github.com/armPelionEdge/maestroSpecs/networking.go

```diff
@@ -70,6 +70,13 @@ type NetIfConfigPayload struct {
 	// WiFi settings, if any
 	WiFiSettings *WiFiSettings `yaml:"wifi" json:"wifi" netgroup:"wifi"`
 
+	// Type of connection.  e.g. wifi or lte
+	Type string `yaml:"type" json:"type"`
```

Comment:
We will need an `lte` netgroup eventually for these parameters to get passed through devicedb